bitnami/elasticsearch/templates/coordinating/statefulset.yaml

```diff
@@ -6,20 +7,25 @@ metadata:
     app.kubernetes.io/component: coordinating-only
     ## Istio Labels: https://istio.io/docs/ops/deployment/requirements/
     app: coordinating-only
-spec:
-  updateStrategy:
-    type: {{ .Values.coordinating.updateStrategy.type }}
-    {{- if (eq "Recreate" .Values.coordinating.updateStrategy.type) }}
-    rollingUpdate: null
+    {{- if .Values.commonLabels }}
+    {{- include "common.tplvalues.render" ( dict "value" .Values.commonLabels "context" $ ) | nindent 4 }}
     {{- end }}
+  {{- if .Values.commonAnnotations }}
+  annotations: {{- include "common.tplvalues.render" ( dict "value" .Values.commonAnnotations "context" $ ) | nindent 4 }}
+  {{- end }}
+spec:
+  {{- if not .Values.coordinating.autoscaling.enabled }}
+  replicas: {{ .Values.coordinating.replicaCount }}
+  {{- end }}
+  podManagementPolicy: Parallel
```

Comment:
I missed that one. But I would like to take this change to ask your opinion on this design matter.
Do you recommend having `.Values.*.podManagementPolicy=Parallel`, or should we continue hardcoding it?
For example, for master nodes setting a value different to `Parallel` could cause the cluster to never start (Because they won't become ready until quorum is met).
For the rest of the roles, not that problematic, but simply would make deployment slower.

Reply:
I'd allow configuring it, setting the default to Parallel